common/pom.xml

```diff
@@ -82,7 +82,11 @@
             <version>${mockserver.version}</version>
             <scope>test</scope>
         </dependency>
-
+        <dependency>
```

Comment:
Any specific reason for this okhttp3? We have so many http clients in our code it is hard to keep track of. Also shouldn't we set the version in the parent pom and then import it?

Reply:
It's required by the slack client library and there is another version of the library that is used in other places so had conflicts.